tsconfig.build.json

```diff
@@ -5,7 +5,7 @@
     "@tsconfig/esm/tsconfig"
   ],
   "compilerOptions": {
-    "declaration": true,
+    "declaration": false,
```

Comment:
Between `1.1.5` and `1.2.0`, the bundle size increased from `48.6 kB` to `62.1 kB`. Since a codemod is supposed to be run, I don't think there's a need to publish declaration files.